doc/changes/latest.inc

```diff
@@ -31,4 +31,8 @@ Bugs
 
 API changes
 ~~~~~~~~~~~
-- None yet
+- Add the argument ``max_iter`` to :func:`~mne.time_frequency.psd_array_multitaper` and to :func:`~mne.time_frequency.csd_array_multitaper` to increase the maximum number of iterations to reach convergence when using ``adaptive=True`` (:gh:`11237` by `Mathieu Scheltienne`_)
+
+Authors
+~~~~~~~
+* Mathieu Scheltienne
```

Comment:
this is not needed. It's done automatically at the end from git log when we release.